src/webui/components/mcp_server_config_tab.py

```
@@ -0,0 +1,154 @@
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)